collections/_article/dev-snapshot-godot-4-1-dev-3.md

```diff
@@ -0,0 +1,139 @@
+- Import: Add support for non-standard OBJ vertex entries ([GH-77042](https://github.com/godotengine/godot/pull/77042)).
+- Import: Fix GLTFSkin binding for the godot_skin property ([GH-77413](https://github.com/godotengine/godot/pull/77413)).
+- Input: Propagate shortcut events to `SubViewport`s ([GH-76926](https://github.com/godotengine/godot/pull/76926)).
+- Input: Fix input ANR in the Godot Android editor ([GH-76980](https://github.com/godotengine/godot/pull/76980)).
+- Input: Add support for DPAD Center key of Android TV remote controller ([GH-77115](https://github.com/godotengine/godot/pull/77115)).
+- Multiplayer: Implement "watched" properties (reliable/on change) ([GH-75467](https://github.com/godotengine/godot/pull/75467)).
+- Navigation: Rework Navigation avoidance ([GH-69988](https://github.com/godotengine/godot/pull/69988)).
+- Navigation: Make navigation mesh edge connections optional ([GH-75601](https://github.com/godotengine/godot/pull/75601)).
+- Navigation: Prevent unnecessary navigation map synchronizations ([GH-75678](https://github.com/godotengine/godot/pull/75678)).
+- Network: Add support for platform-specific CA bundles ([GH-76836](https://github.com/godotengine/godot/pull/76836)).
+- Network: mbedTLS: disable weak crypto and TLS versions ([GH-76905](https://github.com/godotengine/godot/pull/76905)).
+- Network: certs: Sync with Mozilla bundle as of Mar 23, 2023 ([GH-76952](https://github.com/godotengine/godot/pull/76952)).
+- Particles: Rework particle turbulence ([GH-77154](https://github.com/godotengine/godot/pull/77154)).
+- Physics: Restore edge and face support thresholds in 3D ([GH-77171](https://github.com/godotengine/godot/pull/77171)).
+- Porting: Fix 'linux' and specific BSD feature tags ([GH-76974](https://github.com/godotengine/godot/pull/76974)).
+- Porting: Add `audio/general/text_to_speech` project setting to enable/disable TTS. ([GH-77132](https://github.com/godotengine/godot/pull/77132)).
+- Rendering: Fix `draw_multiline_colors` usage ([GH-76989](https://github.com/godotengine/godot/pull/76989)).
+- Rendering: Fix modulation propagation for Y-sorted `CanvasItem`s ([GH-77079](https://github.com/godotengine/godot/pull/77079)).
+- Rendering: Fix LightmapGI dynamic object lighting ([GH-77089](https://github.com/godotengine/godot/pull/77089)).
+- Rendering: (Re-)Implement `Light3D.shadow_reverse_cull_face` ([GH-77238](https://github.com/godotengine/godot/pull/77238)).
+- Rendering: Fix calculation of skinned AABB for unused bones ([GH-77265](https://github.com/godotengine/godot/pull/77265)).
+- Rendering: Take 3D resolution scaling into account for mesh LOD ([GH-77294](https://github.com/godotengine/godot/pull/77294)).
+- Rendering: Fix various issues with blend modes in the OpenGL 3 renderer ([GH-77409](https://github.com/godotengine/godot/pull/77409)).
+- Thirdparty: glad: Re-generate files with glad 2.0.4 ([GH-77350](https://github.com/godotengine/godot/pull/77350)).
```

Comment:
This doesn't change anything (it literally just adds an empty line in a section with already 15 consecutive empty lines :P) so I wouldn't list it.